modules/custom/az_event/templates/az-event-calendar-add-button.html.twig

```diff
@@ -7,7 +7,15 @@
 */
 #}
 {% set attributes = attributes.addClass('clear-both') %}
-{% set links = calendar_links(title, date(start_date), date(end_date), FALSE, description, location) %}
+{# List and order of calendar implementations. #}
+{% set calendar_types = {
+    'google' : 'Google',
+    'ics' : 'iCal File',
+    'webOffice': 'Outlook Enterprise (Office365)',
```

Comment:
```suggestion
    'webOffice': 'Outlook Enterprise (Office 365)',
```

I think I originally suggested `Office365` but I think that should actually include a space.